### PR TITLE
cordova-android 0.9.0

### PR DESCRIPTION
**Pull request by donvigo**
This step adds support for cordova-based Android. It requires [vgaidarji/docker-android-cordova](https://hub.docker.com/r/vgaidarji/docker-android-cordova/) docker image.

### CODE REVIEW

**Comment by viktorbenei** [comment]
Thanks so much, we'll do the review on this step on Monday, ASAP! (The process is always a bit longer for a brand new step).

File: steps/cordova-android/0.9.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+type_tags:
+- cordova
+- build
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+inputs:
+- cordova_command: build
+  opts:
+    is_expand: true
```
Comment (godrei) [inline steps/cordova-android/0.9.0/step.yml]: is_expand: true is the default value of is_expand, so you can remove every is_expand: true.

File: steps/cordova-android/0.9.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    title: Cordova command to run.
+- opts:
+    description: |
+      Choose which platform to build your dependencies for.
+    is_expand: true
+    is_required: true
+    title: Platform.
+    value_options:
+    - android
+  platform: android
```
Comment (godrei) [inline steps/cordova-android/0.9.0/step.yml]: If android is the only one acceptable value_option, maybe you should not wire it out as setp input.

File: steps/cordova-android/0.9.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- build
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+inputs:
+- cordova_command: build
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Run `cordova --help` to check list of available commands.
```
Comment (godrei) [inline steps/cordova-android/0.9.0/step.yml]: Only the steps description will appear on bitrise.io, so i recommand you to provide description for every input.

File: steps/cordova-android/0.9.0/step.yml (hunk 2 above)
Comment (donvigo) [inline steps/cordova-android/0.9.0/step.yml]: I thought about that, currently only Android projects supported, but `cordova` can build different types of project. That's why I decided to keep it.

File: steps/cordova-android/0.9.0/step.yml (hunk 1 above)
Comment (donvigo) [inline steps/cordova-android/0.9.0/step.yml]: Ok, thanks.

File: steps/cordova-android/0.9.0/step.yml (hunk 3 above)
Comment (donvigo) [inline steps/cordova-android/0.9.0/step.yml]: Ok, thanks, I'll fix it.

File: steps/cordova-android/0.9.0/step.yml (hunk 2 above)
Comment (viktorbenei) [inline steps/cordova-android/0.9.0/step.yml]: Makes sense, you can keep this option then :)

**Comment by viktorbenei** [comment]
Sorry, busy day. We'll come back tomorrow - so far we think that the Step can be accepted after the suggested changes, and the docker image should be transferred to https://github.com/bitrise-docker or should be merged into the android image (we're still discussing whether we should merge it or not).

**Comment by donvigo** [comment]
It's ok, no hurry anyway, thank you ;)
I've implemented suggested changes. :rocket:

**Comment by viktorbenei** [comment]
Perfect, thanks for the changes! One last request, if you have the time, could you please squash the commits into one commit? Other than that, everything looks fine ;)

**Comment by donvigo** [comment]
Yes, sure, good point :)
I'll do it once I reach the home in the evening.

**Comment by donvigo** [comment]
Squashed :dancer:

**Comment by viktorbenei** [comment]
Merged & accepted 😎

Thanks for the contribution!
We'll definitely contact you when we (finally) decide what we should do with the docker image 😉

**Comment by donvigo** [comment]
wohooo, thanks :tada:

**Comment by viktorbenei** [comment]
After some discussion we think that these pre-installed tools would be great to have in the default Android image. The size difference is not that significant and it'd make it much easier to setup a new project which requires Cordova.

Would you be open to add the required tools to the base & android docker images, and create a pull request? If you don't have the time that's not an issue either, we'll do that a bit later then.

**Comment by donvigo** [comment]
Let's do in this way: who will have time first, will create this PR. Let's the competition begin :dancers: 
But to be honest, I'll lose to you, as I'm working on a blog and "Cordova Android" project setup guide :)

**Comment by viktorbenei** [comment]
Sounds fantastic! All right, thank you, I'll create a ticket for ourself, and let the battle begin! 😂

**Comment by vgaidarji** [comment]
Please check ["How to build Cordova-based Android project on bitrise.io"](http://vgaidarji.github.io/blog/2016/02/27/how-to-build-cordova-based-android-project-on-bitrise-io/), and if it fits (I hope it does), than feel free to add to bitrise blog ;)

**Pull request merged**